go/cmd/vtexplain/vtexplain.go

```diff
@@ -113,7 +113,6 @@ func getFileParam(flag, flagFile, name string, required bool) (string, error) {
 }
 
 func main() {
-	defer vtexplain.Stop()
```

Comment:
Shouldn't we call vte.Stop() or some other equivalent of this in the end?

Reply:
We are. I just moved the defer to after we have done the Init